### PR TITLE
Refactored member skills to use global exception

### PR DESCRIPTION
**Pull request by moshirim**

### CODE REVIEW

File: server/src/main/java/com/objectcomputing/checkins/services/member_skill/MemberSkillController.java [hunk 1]
```diff
@@ -56,25 +58,16 @@ public MemberSkillController(MemberSkillServices memberSkillServices,
...
-                .link(Link.SELF, Link.of(request.getUri()));
-
-        return HttpResponse.<JsonError>badRequest()
-                .body(error);
-    }
-
-    @Error(exception = MemberSkillAlreadyExistsException.class)
-    public HttpResponse<?> handleAlreadyExists(HttpRequest<?> request, MemberSkillAlreadyExistsException e) {
+    @Error(exception = AlreadyExistsException.class)
+    public HttpResponse<?> handleAlreadyExists(HttpRequest<?> request, AlreadyExistsException e) {
```
Comment (hwilliams314) [inline server/src/main/java/com/objectcomputing/checkins/services/member_skill/MemberSkillController.java:62]: Don't think we need this. Should be handled by the global handler AlreadyExistsHandler in the exceptions/handlers folder

File: server/src/main/java/com/objectcomputing/checkins/services/member_skill/MemberSkillController.java [hunk 2]
```diff
@@ -56,25 +58,16 @@ public MemberSkillController(MemberSkillServices memberSkillServices,
...
         JsonError error = new JsonError(e.getMessage())
                 .link(Link.SELF, Link.of(request.getUri()));
 
         return HttpResponse.<JsonError>status(HttpStatus.CONFLICT).body(error);
     }
 
-    @Error(exception = MemberSkillNotFoundException.class)
-    public HttpResponse<?> handleNotFound(HttpRequest<?> request, MemberSkillNotFoundException e) {
+    @Error(exception = NotFoundException.class)
+    public HttpResponse<?> handleNotFound(HttpRequest<?> request, NotFoundException e) {
```
Comment (hwilliams314) [inline server/src/main/java/com/objectcomputing/checkins/services/member_skill/MemberSkillController.java:70]: Don't think we need this. Should be handled by the global handler NotFoundHandler in the exceptions/handlers folder

Comment (mkimberlin) [reply server/src/main/java/com/objectcomputing/checkins/services/member_skill/MemberSkillController.java:70]: @moshirim Holly is correct. This handler can be removed.

Comment (mkimberlin) [reply server/src/main/java/com/objectcomputing/checkins/services/member_skill/MemberSkillController.java:70]: The same is true for the AlreadyExistsException handler, as Holly mentioned.

File: server/testFile.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+This.Is.A.Test.File
```
Comment (mkimberlin) [inline server/testFile.txt]: Deleted this...it doesn't belong.

**Review by hwilliams314 — APPROVED** [review]

**Review by Bagurp — APPROVED** [review]

**Pull request merged**